File: Smartsense.Mobile.Android/app/src/main/java/com/smartsense/covid/api/model/responses/GetHospitalResponse.java
package com.smartsense.covid.api.model.responses;

import com.smartsense.covid.api.model.Hospital;
import com.smartsense.covid.api.model.VitalHistory;

import java.util.ArrayList;
import java.util.List;

public class GetHospitalResponse extends BaseApiResponse{

    public List<Hospital> hospitals;

    public GetHospitalResponse(List<String> errors) {
        super(errors);
        hospitals = new ArrayList<>();
    }
}

File: Smartsense.Backend/Smartsense.Entity/DTO/GetHospitalResponse.cs
using System;
using System.Collections.Generic;
using System.Text;
using Smartsense.Entity.Models;

namespace Smartsense.Entity.DTO
{
    public class GetHospitalResponse:BaseApiResponse
    {
        public List<Hospital> Hospitals { get; set; }

        public GetHospitalResponse()
        {
            this.Hospitals = new List<Hospital>();
        }
    }
}
